<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<?pydev-testrun version="1.0"?><pydev-testsuite>
    <summary errors="0" failures="0" finished="true" name="Test Run: 0" tests="2" total_time="Finished in: 0.00 secs."/>
    <test location="/Users/jeff/Documents/workspaces/dme/track_logger_converter/tests/converter/data_state_test.py" status="ok" test="Test.test_get_data_names" time="0.00">
        <stdout/>
        <stderr/>
    </test>
    <test location="/Users/jeff/Documents/workspaces/dme/track_logger_converter/tests/converter/data_state_test.py" status="ok" test="Test.test_set_data_item" time="0.00">
        <stdout/>
        <stderr/>
    </test>
    <launch mode="run">
        <launch_memento>
            <![CDATA[<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<launchConfiguration local="true" path="track_logger_converter data_state_test.py"/>
]]>
        </launch_memento>
    </launch>
</pydev-testsuite>
